src/format/parse.rs

```diff
@@ -634,6 +634,9 @@ fn test_parse() {
     check!("",    [lit!("a")]; TOO_SHORT);
     check!(" ",   [lit!("a")]; INVALID);
     check!("a",   [lit!("a")]; );
+    check!("+",   [lit!("+")]; );
+    check!("-",   [lit!("-")]; );
+    check!("−",   [lit!("−")]; );
```

Comment:
is it worth using a unicode escape here to make it clear that this is the unicode minus sign `"\u{2212}"`? I feel like otherwise this could be subtly deleted later

Reply:
I tried using `\u{2212}` but I found code became difficult to grok.
I code commented all `'−'` with some variation of `// MINUS SIGN (U+2212)`.

If you'd strongly prefer `\u{2212}` then I can change it.  But I found the code comment and "native" `−` easier to grok.